admin/views/service-manager/annul-pupil.ejs

```diff
@@ -0,0 +1,41 @@
+<%- partial('partials/_gds_breadcrumbs_and_sign_out', { breadcrumbs }) %>
+<main class="govuk-main-wrapper govuk-body" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">Confirm Pupil Annulment</h1>
+            <%- partial('partials/_gds_session_expiry') %>
+            <%- partial('partials/_flash-messages') %>
+            <%- partial('partials/_form-error', {
+              pageFields: ['upn'],
+              error
+          }) %>
+
+            <form action="" method="post">
+                <input type="hidden" name="_csrf" value="<%= csrftoken %>"/>
+                <div class="govuk-form-group">
+                    <label class="govuk-label" for="upn">
+                        Confirm UPN for <%=pupil.lastName %>,<%=pupil.firstName %>
```

Comment:
How does this work if the UPN is not unique? I was expecting a pupil urlSlug...

Reply:
regards uniqueness of UPN - the UX journey has you search via UPN, you choose the pupil you want to view in the summary, then you choose to annul (all guided by the URL slug), and then you simply confirm the UPN.  its not used in the selection during annulment, just to confirm the UPN is the same as the one you just viewed